### PR TITLE
Port Play Tests to Er Force Sim

### PR DESCRIPTION
**Pull request by jonathanlew**
### Description
This PR ports all the play tests to the Er Force Simulator that will pass in their current state

### Resolved Issues
resolves #2422 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #include "software/simulated_tests/simulated_er_force_sim_test_fixture.h"
 
+// TODO (#2419): remove this
+#include <fenv.h>
```
Comment (Joshuayellowley) [inline src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp:4]: Is this being added to handle the SIGFPE errors we saw previously in the kickoff_friendly_er_force_test?

**Review by Joshuayellowley — APPROVED** [review]
Just one small question, but the changes LGTM!

**Review by PhilD71 — APPROVED** [review]
Awesome!

File: src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp (hunk 1 above)
Comment (jonathanlew) [reply src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp:4]: yes, we disable sigfpe when the er force simulator is running

**Review by akhilveeraghanta — APPROVED** [review]
lgtm!

**Pull request merged**